Library/Homebrew/dev-cmd/bump.rb

```diff
@@ -277,8 +277,9 @@ def retrieve_pull_requests(formula_or_cask, name, version: nil)
           odebug "Error fetching pull requests for #{formula_or_cask} #{name}: #{e}"
           nil
         end
+        return if pull_requests.blank?
```

Comment:
The method was returning an empty string when there are no pull requests, so this guard ensures that the method returns `nil` when there are no pull requests. This allows us to simplify conditional checks for the return value (e.g., omitting `presence` calls).